account_payment_unece/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+
+{
+    "name": "Account Payment UNECE",
+    "version": "18.0.1.0.0",
+    "category": "Accounting & Finance",
+    "license": "AGPL-3",
+    "summary": "UNECE nomenclature for the payment methods",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "maintainers": ["alexis-via"],
+    "website": "https://github.com/OCA/community-data-files",
+    "depends": ["account_payment_base_oca", "base_unece"],
```

Comment:
Do we really need the dependency on `account_payment_base_oca`?

Reply:
The dependency is ONLY for the views. There is no form/list view of account.payment.method in the "account" module.

Reply:
Updated, see my comments below